json_test.go

```diff
@@ -0,0 +1,29 @@
+package errors
+
+import (
+	"encoding/json"
+	"regexp"
+	"testing"
+)
+
+func TestFrameMarshalJSON(t *testing.T) {
+	var tests = []struct {
+		Frame
+		want string
+	}{{
+		initpc,
+		`^"github.com/pkg/errors.init(.ializers)? .+/github.com/pkg/errors/stack_test.go:\d+"$`,
```

Comment:
Please escape periods that must match periods.

Reply:
I just copied then adapted these REs from existing tests, but will do.

Reply:
Indeed, it’s super common to miss, and rarely does it ever cause any problems.

Also, you missed one… :wink:

Reply:
Fixed that one now, too.